lib/awspec/helper/finder/apigateway.rb

```diff
@@ -2,17 +2,23 @@ module Awspec::Helper
   module Finder
     module Apigateway
       def find_apigateway_by_id(id)
-        rest_apis = apigateway_client.get_rest_apis
-        rest_apis.items.each do |item|
-          return item if item.id == id
+        apis = []
+        apigateway_client.get_rest_apis(limit: 500).each do |response|
```

Comment:
Thank you for your commit !!!

Could you use `position:` for paging ? ( see https://docs.aws.amazon.com/sdk-for-ruby/v3/api/Aws/APIGateway/Client.html#get_rest_apis-instance_method )

Reply:
Hey k1LoW, thank you for reviewing my PR! AWS actually made responses enumerable so this is using the `position:` param even though it doesn't say it explicitly.  I am iterating through the `position`s.  This is the standard for using AWS pagination as defined in the documentation here:  https://docs.aws.amazon.com/sdk-for-ruby/v3/developer-guide/paging-responses.html

Reply:
Oops! You are right!